src/components/footer.jsx

```diff
@@ -48,56 +48,58 @@ function Footer() {
   const secondaryLinks = links.secondary;
 
   return (
-    <FooterUSWDS
-      size="medium"
-      // TODO: for some reason class name has to be manually added for returnToTop
-      returnToTop={(
-        <GridContainer>
-          <Grid className="usa-footer__return-to-top">
-            <Link to="#top">
-              {t('footer.returnToTop')}
-            </Link>
-          </Grid>
-          <div id="feedback-widget" className="footer__feedback-widget" lang={i18n.language} pageTitle={t('title')} endpoint={process.env.GATSBY_ENDPOINT} />
-          <ScriptTag src="https://d2ttz3as5y3dj0.cloudfront.net/feedback-module.min.js" type="text/javascript" />
-        </GridContainer>
-      )}
-      primary={(
-        <FooterNav
-          size="medium"
-          links={primaryLinkItems}
+    <div>
+      <div className="footer__return-to-top-feedback-container">
+        <Grid className="usa-footer__return-to-top">
+          <Link to="#top">{t('footer.returnToTop')}</Link>
+        </Grid>
+        <div
+          id="feedback-widget"
+          className="footer__feedback-widget"
+          lang={i18n.language}
+          pageTitle={t('title')}
```

Comment:
Are we going to keep the default title of the project instead of the specific page title?

Reply:
Not sure - I thought that project title might be better than page title. Because if the page title is "Project | Home" or something along those lines it reads kind of weirdly in a sentence. What do you think?

Reply:
Yes, let's keep the project title.